pkg/api/bazeldnf/config.go

```diff
@@ -0,0 +1,12 @@
+package bazeldnf
+
+type RPM struct {
+	Name string `json:"name"`
+	SHA256 string `json:"sha256"`
+	URLs []string `json:"urls"`
+}
+
+type Config struct {
```

Comment:
Should we have a lock file version field? Or we add that one when we start adding extra fields?

Reply:
Same, we add this in the future version